fp.go

```diff
@@ -121,6 +121,63 @@ func (x *Fp) PutInt(dst *big.Int) {
 	dst.SetBytes(bs[:])
 }
 
+// PutB32 stores the bytes of the field element into destination in big endian
+// form.
+//
+// Panics: If the byte slice has length less than 32, this function will panic.
+func (x Fp) PutB32(dst []byte) {
+	if len(dst) < 32 {
+		panic(fmt.Sprintf("invalid slice length: length needs to be at least 32, got %v", len(dst)))
+	}
+
+	// NOTE: This function assumes that the representation is normalised.
```

Comment:
Consider putting this note in the comments that document the function. That way it will show up in godoc and IDEs.

Reply:
I put the comment here because it is this `putB32From5x52` function that has this requirement, and also the design philosophy for this type was for the user to not have to know about normalisation, nor be able to create instances of `Fp` that are not normalised.

Reply:
As long as it's not something the caller of this function needs to be actively aware of, then that is fine.